src/common.ts

```diff
@@ -154,13 +154,25 @@ export class LoggingCommon {
       entryMetadata.httpRequest = metadata.httpRequest;
     }
 
+    // If running on Google Cloud Functions, add labels for the function
+    // name, project, and region.
```

Comment:
Q: We are special casing cloud functions here. Should we also be doing something for GAE and other managed environments? Note that we already do provide some of this information in the `resource` field of the LogEntry: https://github.com/googleapis/nodejs-logging/blob/34878be64cb4475e364624c5a5ea86d4f809fcff/src/log.ts#L825-L834

Reply:
Yes, providing info for GAE would be good too.  I was thinking that could be a follow-up PR, but if you think it belongs in this PR, I can add it.

Reply:
It would be good to know what kind of logic we need to add. Secondly, is there a way to put this logic in the common logging library rather than here? I expect the `logging-bunyan` also needs it.

Reply:
Yes, I am still investigating what information we should auto-populate from GAE.  Yes, putting this in `@google-cloud/logging` would be better.  I'll move this work there.